### PR TITLE
Adding /time

### PR DESCRIPTION
**Pull request by WoofHax**

### CODE REVIEW

**Comment by moanhawks** [comment]
me forget 1 things to add #include <ctime>

**Comment by ZentaiZber** [comment]
adding uptime lel is ez

**Comment by Deveytv** [comment]
@moanhawks you need to fix that, before make it ps 
image

**Comment by moanhawks** [comment]
#include <<ctime>>

**Comment by herothedev** [comment]
Who doesnt have this added lol 
`#pragma warning(disable:4996)`

**Comment by moanhawks** [comment]
I think this method can make system ban/curse/mute time

**Comment by ZentaiZber** [comment]
yes if you pro on coding maybe like bantime = timeend -> bantime - currenttime

File: enet server test/enet server test.cpp [hunk 1]
```diff
@@ -3178,6 +3194,10 @@ int _tmain(int argc, _TCHAR* argv[])
 						data.plantingTree = 0x0;
 						SendPacketRaw(4, packPlayerMoving(&data), 56, 0, peer, ENET_PACKET_FLAG_RELIABLE);*/
 					}
+					else if (str.substr(0, 5) == "/time")
```
Comment (OverallGod) [inline enet server test/enet server test.cpp]: Fix this please 
it should be ```else if (str == "/time") {```
but it is  ```else if (str.substr(0, 5) == "/time"){```

**Comment by flossboss** [comment]
Also it says:
```cppGamePacket p = packetEnd(appendString(appendString(createPacket(), "OnConsoleMessage"), "`2Today date is: " + test));``` but shouldn't it be ```GamePacket p = packetEnd(appendString(appendString(createPacket(), "OnConsoleMessage"), "`2Current time is: " + test));```

**Comment by moanhawks** [comment]
Sorry If my English bad

**Pull request closed without merging**